runtime/web_worker.rs

```diff
@@ -440,7 +441,7 @@ impl WebWorker {
       deno_napi::deno_napi::init_ops::<PermissionsContainer>(),
       deno_http::deno_http::init_ops(),
       deno_io::deno_io::init_ops(Some(options.stdio)),
-      deno_fs::deno_fs::init_ops::<PermissionsContainer>(unstable),
+      deno_fs::deno_fs::init_ops::<_, PermissionsContainer>(unstable, StdFs),
```

Comment:
Why do you need to pass `StdFs` as argument to the function call and not as type param before `PermissionsContainer`?

Reply:
`StdFs` is the object not the trait type. First type param is the trait of StdFs

Reply:
Exactly, in the case of a VFS, the `StdFs` would be the instance of a struct, instead of a 0-size type.

Reply:
Got it, was just curious, because I had no need to store state in case of `NodeEnv::Fs` and it was only passed as a type param.